src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -42,6 +42,7 @@
                       </result>
                     </resultlist>')
   end
+  let(:branched_project_name) { ([source_project.name] * 2).join(':branches:') }
```

Comment:
Why do you calculate this? It will never change right?

Reply:
@hennevogel 

> Why do you calculate this? It will never change right?

It will never change. What is your alternative?

Reply:
Do not calculate it? Wherever you used `branched_project_name` use 

```ruby
#{source_project.name}:branches:#{source_project.name}
```

then I don't have to look up what this let is whenever I read this code...

Reply:
@hennevogel Changed. Please re-review.